components/layout_2020/lists.rs

```diff
@@ -55,11 +55,11 @@ where
 fn marker_string(style: &style_structs::List) -> Option<&'static str> {
     match style.list_style_type {
         ListStyleType::None => None,
-        ListStyleType::Disc => Some("• "),
-        ListStyleType::Circle => Some("◦ "),
-        ListStyleType::Square => Some("▪ "),
-        ListStyleType::DisclosureOpen => Some("▾ "),
-        ListStyleType::DisclosureClosed => Some("‣ "),
+        ListStyleType::Disc => Some("•\u{00a0}"),
+        ListStyleType::Circle => Some("◦\u{00a0}"),
+        ListStyleType::Square => Some("▪\u{00a0}"),
+        ListStyleType::DisclosureOpen => Some("▾\u{00a0}"),
+        ListStyleType::DisclosureClosed => Some("‣\u{00a0}"),
```

Comment:
This seems like it would cause extra spacing for inside markers, in a way that would match Firefox but was considered bad by the CSSWG, see https://github.com/w3c/csswg-drafts/issues/4891.

But it doesn't actually seem to happen, which may be due to some white space processing bug? So I guess it's fine for now, but we should consider setting `white-space: pre` just for outside markers.

Reply:
Not necessarily, because it should only be added to outside markers, and there is no `::marker:outside` or such.